packages/edge/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@llamaindex/edge",
+  "version": "0.1.21",
+  "license": "MIT",
+  "type": "module",
+  "dependencies": {
+    "@anthropic-ai/sdk": "^0.15.0",
+    "@aws-crypto/sha256-js": "^5.2.0",
+    "@datastax/astra-db-ts": "^0.1.4",
+    "@llamaindex/cloud": "0.0.4",
+    "@llamaindex/env": "workspace:*",
+    "@mistralai/mistralai": "^0.0.10",
+    "@notionhq/client": "^2.2.14",
+    "@pinecone-database/pinecone": "^2.0.1",
+    "@qdrant/js-client-rest": "^1.7.0",
+    "@types/lodash": "^4.14.202",
+    "@types/node": "^18.19.14",
+    "@types/papaparse": "^5.3.14",
+    "@types/pg": "^8.11.0",
+    "@xenova/transformers": "^2.15.0",
+    "assemblyai": "^4.2.2",
+    "chromadb": "~1.7.3",
+    "cohere-ai": "^7.7.5",
+    "js-tiktoken": "^1.0.10",
+    "lodash": "^4.17.21",
+    "magic-bytes.js": "^1.10.0",
+    "mammoth": "^1.6.0",
+    "md-utils-ts": "^2.0.0",
+    "mongodb": "^6.3.0",
+    "notion-md-crawler": "^0.0.2",
+    "openai": "^4.26.1",
+    "papaparse": "^5.4.1",
+    "pathe": "^1.1.2",
+    "pdf2json": "^3.0.5",
+    "pg": "^8.11.3",
+    "pgvector": "^0.1.7",
+    "portkey-ai": "^0.1.16",
+    "rake-modified": "^1.0.8",
+    "replicate": "^0.25.2",
+    "string-strip-html": "^13.4.6",
+    "wink-nlp": "^1.14.3"
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "types": "./dist/type/index.edge.d.ts",
+  "main": "./dist/index.edge.js",
+  "exports": {
+    "./readers/SimpleDirectoryReader": {
+      "import": {
+        "types": "./dist/type/readers/SimpleDirectoryReader.edge.d.ts",
```

Comment:
We could call this EdgeDirectoryReader then we won't need this edge specific export